Traceback (most recent call last):
  File "C:\Users\pmd19\Anaconda3\envs\jb\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "C:\Users\pmd19\Anaconda3\envs\jb\lib\site-packages\nbclient\client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\pmd19\Anaconda3\envs\jb\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\pmd19\Anaconda3\envs\jb\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\pmd19\Anaconda3\envs\jb\lib\asyncio\base_events.py", line 583, in run_until_complete
    return future.result()
  File "C:\Users\pmd19\Anaconda3\envs\jb\lib\site-packages\nbclient\client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "C:\Users\pmd19\Anaconda3\envs\jb\lib\site-packages\nbclient\client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Users\pmd19\Anaconda3\envs\jb\lib\site-packages\nbclient\client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Import all functions using a name
import numpy as np
# We then use the name to refer to functions from this module
print( np.sin( 1./2. * np.pi ) )

# We can also import just some of the functions, as well as change their names
from math import cos as mathCos
print( mathCos( np.pi ) )
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_16424/1140472914.py[0m in [0;36m<module>[1;34m[0m
[0;32m      1[0m [1;31m# Import all functions using a name[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 2[1;33m [1;32mimport[0m [0mnumpy[0m [1;32mas[0m [0mnp[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      3[0m [1;31m# We then use the name to refer to functions from this module[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[0;32m      4[0m [0mprint[0m[1;33m([0m [0mnp[0m[1;33m.[0m[0msin[0m[1;33m([0m [1;36m1.[0m[1;33m/[0m[1;36m2.[0m [1;33m*[0m [0mnp[0m[1;33m.[0m[0mpi[0m [1;33m)[0m [1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      5[0m [1;33m[0m[0m

[1;31mModuleNotFoundError[0m: No module named 'numpy'
ModuleNotFoundError: No module named 'numpy'

